packages/salesforce-adapter/src/change_validators/flexi_page_unused_or_missing_facets.ts

```diff
@@ -44,14 +44,14 @@ const getFacetDefinitionsAndReferences = (
   element: InstanceElement,
 ): { facetDefinitions: Record<string, ElemID>; facetReferences: Map<string, ElemID[]> } => {
   const facetDefinitions: Record<string, ElemID> = {}
-  const facetReferences = new DefaultMap<string, ElemID[]>(() => [])
+  const unFilteredFacetReferences = new DefaultMap<string, ElemID[]>(() => [])
```

Comment:
```suggestion
  const unfilteredFacetReferences = new DefaultMap<string, ElemID[]>(() => [])
```
"Unfiltered" is one word.

Reply:
I chose this name because before filtering, there might be references to things that aren’t actually facets. Calling it allFacetReferences could be misleading. Do you think potentialFacetReferences is better?

Reply:
Yeah, let's go with "potential".